app/views/moderations/actions_panel.html.erb

```diff
@@ -56,7 +56,7 @@
         class="c-btn c-btn--primary c-btn--destructive w-100 my-4"
         id="unpublish-article-btn"
         data-article-id="<%= @moderatable.id %>"
-        data-author-username="<%= @moderatable.username %>"
+        data-author-username="<%= author_username %>"
```

Comment:
According to my truncated-project proposal [discussed here](https://forem-team.slack.com/archives/C03CHBBMTJP/p1658500957993839), this Unpublish Post button would be hidden from the Mod Panel when the user is in the Mod Center.

This button is not working in the Mod Center, because it uses a Preact modal for which the content is specific to the article, and is hidden in the individual article’s page. In the Mod Center, we have several articles, and this hidden data is not present, hence the modal fails to load.